pkg/securityscan/job/job.go

```diff
@@ -196,6 +210,12 @@ func New(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile *cisopera
 						}, {
 							Name:      `etc-group`,
 							MountPath: `/etc/group`,
+						}, {
+							Name:      `journalctl`,
+							MountPath: `/var/log/journal`,
+						}, {
+							Name:      `syslog`,
+							MountPath: `/var/log/syslog`,
```

Comment:
same question as Nick's above - I believe some distros use /var/log/messages instead?

Reply:
The process only uses our images which we control.

Reply:
Maybe this can be set via environment variables? Default it to `/var/log/journal` and `/var/log/syslog` but provide a mechanism to override?

Reply:
This will only vary if we decide to change images to something that diverges from what we're currently using. If that's not planned, would making this configurable even be necessary?